pkg/lib/clusterconfig/clusterconfig.go

```diff
@@ -190,6 +191,10 @@ var UserValidation = &cr.StructValidation{
 				AllowedValues: aws.EKSSupportedRegionsSlice,
 			},
 		},
+		{
+			StructField:          "AvailabilityZones",
+			StringListValidation: &cr.StringListValidation{},
```

Comment:
Should we add a validation that blocks the user from setting availability zones to `[]`? There might be a field in configreader for that but I'm not sure